src/components/Subthoughts.tsx

```diff
@@ -364,7 +364,8 @@ export const SubthoughtsComponent = ({
   const state = store.getState()
   const [page, setPage] = useState(1)
   const globalSort = getSetting(state, ['Global Sort']) || 'None'
-  const sortPreference = contextSort || globalSort
+  const rootSort = isRoot(thoughtsRanked) ? attribute(store.getState(), thoughtsRanked!, '=sort') : null
+  const sortPreference = (contextSort || rootSort) || globalSort
```

Comment:
Since `Subthoughts` component gets `sort` as a prop, it won't re-render if we put the `rootSort` logic into `Subthoughts` render function. `Content` is the actually the parent component of the root `Subthoughts`, so we need to place that logic inside `Content` to make it work.